### DISCUSSION TITLE
20241025 - Remove single item from playlist removes ALL items from playlist

### DISCUSSION THREAD

**Discussion by markblinkhorn** [rern/rAudio#1798] [Bug Reports]
Title says it all...

**Comment by EarsofTin** [comment]
Me too. Viewing playlist, select a single track with the note icon, choose "remove" and every track dissapears.

**Comment by EarsofTin** [comment]
Update 20241108 fixes my problem. Thank you much.

**Comment by markblinkhorn** [comment]
Fixed by 20241108. Thank you.
